test/Driver/batch_mode_print_jobs.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: %empty-directory(%t)
+// RUN: touch %t/file-01.swift %t/file-02.swift %t/file-03.swift
+// RUN: echo 'public func main() {}' >%t/main.swift
+//
+// RUN: %swiftc_driver -enable-batch-mode -c -emit-module -module-name main -j 2 %t/file-01.swift %t/file-02.swift %t/file-03.swift %t/main.swift  -driver-print-jobs | %FileCheck %s -check-prefix=CHECK-COMBINED
+// RUN: %swiftc_driver -enable-batch-mode -c -emit-module -module-name main -j 2 %t/file-01.swift %t/file-02.swift %t/file-03.swift %t/main.swift  -###  | %FileCheck %s  -check-prefix=CHECK-COMBINED
+// RUN: %swiftc_driver -enable-batch-mode -driver-print-jobs -c -emit-module -module-name main -j 2 %t/file-01.swift %t/file-02.swift %t/file-03.swift %t/main.swift 2>%t/shouldBeEmpty >/dev/null && test -z `cat %t/shouldBeEmpty`
```

Comment:
Why is this empty?

Reply:
In the implementation, it is important to get -### to output on the expected file descriptor. Many existing tests depend on it. So I test for the right output on the right fd, and for no output on the wrong fd.

Reply:
Other output flags write on the other fd.

Reply:
Ahhh, I missed that. Maybe leave a comment? Or just do one invocation that writes both output streams to separate files, which makes it more obvious that they're being treated differently.